test/tests/functional/pbs_accumulate_resc_used.py

```diff
@@ -1180,13 +1208,15 @@ def test_epi_pro(self):
         jid = self.server.submit(j)
 
         # Verify the resources_used once the job is over
-        self.server.expect(JOB, {
+        b = {
             'resources_used.foo_i': '30',
             'resources_used.foo_f': '0.6',
-            'resources_used.cput': '30',
-            'job_state': 'F'}, attrop=PTL_AND,
-            extend='x', id=jid, offset=5,
-            max_attempts=60, interval=1)
+            'job_state': 'F'}
+
+        if has_cpuset is 0:
+            b['resources_used.cput'] = '30'
+        self.server.expect(JOB, b, attrop=PTL_AND, extend='x', id=jid,
+                           offset=5, max_attempts=60, interval=1)
```

Comment:
You can drop the PTL_AND.  It is the default now.  Also max_attempts=60 is default as well.

Reply:
Okay, this was there before, I just moved the lines up.  But I can remove these now that they are the default.